freqtrade/freqai/prediction_models/XGBoostRFRegressor.py

```diff
@@ -44,7 +45,10 @@ def fit(self, data_dictionary: Dict, dk: FreqaiDataKitchen, **kwargs) -> Any:
 
         model = XGBRFRegressor(**self.model_training_parameters)
 
+        model.set_params(callbacks=[TBCallback(dk.data_path)], activate=self.activate_tensorboard)
```

Comment:
Is this change related to pipelines? (all changes in this file, really)?

if yes, then i think i misunderstood how pipelines work . . .

Reply:
No, just tossed it in there when I noticed it. It is just adding tensorboard to XGBRFRegressor.